prepare_models.py

```diff
@@ -5,7 +5,7 @@
 
 conf = Configuration()
 
-
+@staticmethod
```

Comment:
I don't get all these `@staticmethod`s that have been added. I am not requesting to revert them, but it's generally not a good practice to apply changes like these in a "new feature" pull request.

Reply:
I put them in the beginning, when trying to figure out how the code worked. They will all be removed